packages/angular/projects/angular/src/internal/dynamic-template/dynamic-template.component.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ComponentRef,
+  EmbeddedViewRef,
+  Input,
+  TemplateRef,
+  ViewContainerRef,
+} from '@angular/core';
+import { UnknownRecord } from '@devextreme/core';
+import { AngularTemplate } from '../types';
+import { ViewComponent } from './view.component';
+
+interface AngularTemplateContext<T> {
+  $implicit: T;
```

Comment:
What is '$implicit' (as I undestand it props for push data to template)? Is it standard Angular template props? Can you provide some doc ref? (I try but without success)

Reply:
Unfortunately, the Angular templates are badly documented.

```
Using the key $implicit in the context object will set its value as default.
```
[Here is the link to the docs with $implicit](https://angular.io/api/common/NgTemplateOutlet#description)

Our clients may don't know what name of the variable we pass to the template.
And with ```$implicit``` usage they don't need to think about it, they can just declare the template variable.

```html
<!-- template without $implicit -->
<ng-template let-data="variable_name"> <!-- <-- need to know what to write here -->
  {{ data }}
</ng-template>

<!-- template with $implicit -->
<ng-template let-data> <!-- <-- just declare template variable -->
  {{ data }}
</ng-template>